packages/core-database/__tests__/repositories/delegates.test.js

```diff
@@ -1,17 +1,17 @@
 'use strict'
 
 const app = require('../__support__/setup')
-const genesisBlock = require('../__fixtures__/genesisBlock')
-
 const { crypto } = require('@arkecosystem/crypto')
 
+let genesisBlock
 let repository
 let walletManager
 let calculateApproval
 let calculateProductivity
 
 beforeAll(async (done) => {
   await app.setUp()
+  genesisBlock = require('../__fixtures__/genesisBlock')
```

Comment:
Could you add a brief comment here and in the other files? It's the best way to avoid that other devs that aren't aware of these subtle problems, make this same error

Reply:
Done.